x-pack/filebeat/input/azureblobstorage/client.go

```diff
@@ -0,0 +1,102 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+//go:build !aix
+// +build !aix
+
+package azureblobstorage
+
+import (
+	"fmt"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob"
```

Comment:
This import at this version has a go version of go1.18. v0.3.0 would build since it's constrained to go1.16.

Reply:
@efd6 v0.3.0 has many features missing that would make the current implementation of this input unusable , hence I've attached a blocked tagged to this for the moment.

Reply:
No worries, I suspected that was the case, just noting here for the reason for the build failure.